Cargo.toml

```
@@ -7,3 +7,5 @@ authors = ["Gabriel Martinez <reitaka@gmail.com>",
 
 [dependencies]
 ggez = "0.5.0-rc.2"
+specs = "0.15.0"
```

```suggestion
specs = "0.15"
```
I tend to prefer specifying just the major (or minor if the major version is 0) of a crate. Unless if we specifically need a particular version.

Same for the line below :)